lib/taurus/qt/qtcore/util/emitter.py

```diff
@@ -206,7 +252,7 @@ def clear(self):
             self._done += 1
 
     def purge(obj):
```

Comment:
I am confused here: the purge method does not define `self`, yet it uses it.

I've searched and I  do not see the purge method being called at all in Taurus, or it being part of an Qt API,... so I am tempted to simply remove the `purge` method, **but**...

I see the same thing happening in other classes in this same module and this PR actually does some changes in the purge method and uses `self` there, so I am worried that I am missing something... 

Can anyone clarify if this is just buggy leftover code or if somehow it makes sense?